templates/quiz/results-attempt-questions-list.php

```diff
@@ -5,7 +5,8 @@
  * @since 3.16.0
  * @since 3.17.8 Unknown.
  * @since 5.3.0 Display removed questions too.
- * @version 5.3.0
+ * @since [version] Hide answers if resumable attempt is incomplete.
+ * @version [version]
```

Comment:
This is a good occasion to escape all the output that can be escaped :D
can you?

Reply:
yup sure thing.